src/HornedBeast.js

```diff
@@ -1,21 +1,13 @@
 import React from "react";
-import HornedGopher from "./horned-gophers.jpeg"
-import Cassowary from "./cassowary1.jpg"
-import NubianIbex from "./nubianibex.jpeg"
+
 
 class HornedBeast extends React.Component {
   render() {
     return (
       <div>
-        <h2>{this.props.title1}</h2>
-        <img src={HornedGopher} alt="horned gopher-skeleton" title="Horned Gopher"></img>
-        <p>{this.props.description1}</p>
-        <h2>{this.props.title2}</h2>
-        <img src={Cassowary} alt="horned bird" title="Cassowary"></img>
-        <p>{this.props.description2}</p>
-        <h2>{this.props.title3}</h2>
-        <img src={NubianIbex} alt="horned gopher-skeleton" title="Horned Gopher"></img>
-        <p>{this.props.description3}</p>
+        <h2>{this.props.title}</h2>
+        <img src={this.props.imageUrl} alt={this.props.alt}></img>
+        <p>{this.props.description}</p>
       </div>
```

Comment:
Love to see this! You worked on refactoring your code so that you didn't have so much of the same thing over and over again. And now you're hyper modularized, so you can keep adding more and more beasts.